easybuild/easyconfigs/n/Nextflow-pipeline/Nextflow-pipeline-20221110-foss-2021a-mag.eb

```diff
@@ -0,0 +1,49 @@
+##
+# This is a contribution from SIB Swiss Institute of Bioinformatics
+# Homepage:     https://www.sib.swiss/research-infrastructure/competence-centers/vital-it
+#
+# Authors::     Sebastien Moretti <sebastien.moretti@sib.swiss>
+#
+##
+
+easyblock = 'Bundle'
+
+name = 'Nextflow-pipeline'
+version = '20221110'
+versionsuffix = '-mag'
+
+homepage = 'https://nf-co.re/mag'
+description = """The Nextflow pipeline 'mag' ported to EasyBuild/EESSI."""
+
+toolchain = {'name': 'foss', 'version': '2021a'}
+
+# TODO The long reads sequencing part tools will be added later
+dependencies = [
+    ('Nextflow',       '22.10.1', '', SYSTEM),
+    ('AdapterRemoval', '2.3.2'),
+    ('BCFtools',       '1.12'),
+    ('Bowtie2',        '2.4.4'),
+    ('BUSCO',          '5.4.2'),
+    ('CAT-BAT',        '5.2.3'),
+    ('Centrifuge',     '1.0.4'),
+    ('DAS_Tool',       '1.1.3', '', ('foss', '2021a-R-4.1.0')),
```

Comment:
This is not entirely correct, all you want is the `-R-4.1.0` as `versionsuffix`, and there's no need to specify the toolchain:

```python
('DAS_Tool',       '1.1.3', '-R-4.1.0'),